src/main/java/edu/iu/terracotta/utils/TextConstants.java

```diff
@@ -62,5 +62,6 @@ private TextConstants() {
     public static final String ASSIGNMENT_LOCKED = "Error 116: The assignment is not open at this moment";
     public static final String LIMIT_OF_SUBMISSIONS_REACHED ="Error 117: You can't answer this assignment again";
     public static final String MAX_SUBMISSION_ATTEMPTS_REACHED = "Error 150: Max submission attempts already reached";
+    public static final String ASSIGNMENT_SUBMISSION_WAIT_TIME_NOT_REACHED = "Error 151: You must wait to answer this assignment again";
```

Comment:
Can you change the error code number to 158?  We need to manage and document these better, because it's not obvious at all that 151-157 are already taken.

I tend to use `egrep -r "Error \d{3}:" src/main/ | sed -rn 's/.*(Error [0-9]{3}):[[:space:]]*([^"]+).*/\1: \2/gp' | sort -n | uniq` to find already used error codes. See TCOTA-349 for more info.

Reply:
Done!

And thanks for the tip. I was wondering how those error numbers were incremented. :)